js/ui/grid_core/ui.grid_core.column_fixing.js

```diff
@@ -199,15 +199,16 @@ var baseFixedColumns = {
             alignByFixedColumnCellCount,
             column = options.column,
             isFixedTableRendering = that._isFixedTableRendering,
-            isGroupCell = options.rowType === "group" && isDefined(column.groupIndex);
+            isGroupCell = options.rowType === "group" && isDefined(column.groupIndex),
+            isUnfixedButtonColumn = column.type === "buttons" && !column.fixed;
 
         // T747718
         if(isFixedTableRendering && isGroupCell && !column.command) {
             $cell.css("pointerEvents", "none");
         }
 
         if(!isFixedTableRendering && that._isFixedColumns) {
-            isEmptyCell = column.fixed || column.command;
+            isEmptyCell = column.fixed || (column.command && !isUnfixedButtonColumn);
```

Comment:
This is wrong expression. The correct version:

isEmptyCell = column.fixed || (column.command && column.fixed !== false);